examples/c/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(CMAKE_CXX_STANDARD 20)
 option(USE_CUDA "Build with CUDA support" OFF)
 option(USE_CXX "Invoke the C++ example" ON)
 option(PHI3 "Build the Phi example" OFF)
+option(PHI3_QA "Build the Phi Q&A example without memory" OFF)
```

Comment:
Does without memory imply without multi-turn prompt support? Could we replace `without memory` with something more descriptive (without multi-turn prompting or example showcasing non-continuous decoding)